components/compounds/Header.tsx

```diff
@@ -11,35 +11,38 @@ import { TranslatedText } from '../atoms/TranslatedText';
 
 const Menu = React.memo(() => {
   const router = useRouter();
-  const { locale } = useContext(TranslationContext);
+  const { locale, translation } = useContext(TranslationContext);
   // consider extracting to generic routing function
-  const handleButtonPress = (path: string) => {
-    router.push(pathFromComponents(locale, path));
+  const handleButtonPress = (path: string, hash?: string) => {
+    router.push({
+      pathname: pathFromComponents(locale, path),
+      hash: hash
+    });
   };
 
   return (
     <>
       <div className="hidden md:flex flex-row justify-between w-128">
         <LanguagePicker />
-        <Button variant="text" color="secondary" onClick={ () => handleButtonPress('/login') }>
-          Pricing
+        <Button variant="text" color="secondary" onClick={ () => handleButtonPress('/', 'pricing') }>
+          { translation.header.pricing }
         </Button>
         <Button variant="text" color="secondary" onClick={ () => handleButtonPress('/login') }>
-          Contact us
+          { translation.header.contactUs }
         </Button>
         <Button variant="contained" color="secondary" onClick={ () => handleButtonPress('/login') }>
-          Login
+          { translation.header.login }
         </Button>
       </div>
       <div className="md:hidden flex flex-grow justify-end mr-4" >
         <LanguagePicker/>
       </div>
       <DrawerMenu className="md:hidden">
-        <DrawerMenuItem icon={ <LocalOfferOutlined /> } label="Pricing" path='/'/>
-        <DrawerMenuItem icon={ <MailOutlined /> } label="Contact us" path='/'/>
+        <DrawerMenuItem icon={ <LocalOfferOutlined /> } label={ translation.header.pricing } onClick={ async () => handleButtonPress('/', 'pricing') } />
+        <DrawerMenuItem icon={ <MailOutlined /> } label={ translation.header.contactUs } onClick={ async () => handleButtonPress('/login') } />
         <Divider variant="middle" />
-        <DrawerMenuItem icon={ <AddBoxOutlined /> } label="Sign up" path='/'/>
-        <DrawerMenuItem icon={ <ExitToAppOutlined /> } label="Log in" path='/login'/>
+        <DrawerMenuItem icon={ <AddBoxOutlined /> } label={ translation.header.signUp } onClick={ async () => handleButtonPress('/login') } />
+        <DrawerMenuItem icon={ <ExitToAppOutlined /> } label={ translation.header.login } onClick={ async () => handleButtonPress('/login') } />
```

Comment:
Consider changing the prop type in `DrawerMenuItem.tsx` for `onClick` to `() => void`. The current type expects the return of an object which is why Typescript prompts `async` here, because I believe that comes with the implicit return of a promise in this case (no `{}`).

Accordingly, if making that change, consider removing `async`.

Reply:
Ohhhhh right, didn't think of this